test/built-ins/Object/subclass-object-arg.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-object-value
+author: Matthew Phillips <matthew@matthewphillips.info>
+description: >
+  NewTarget is active function and subclass of Object
+info: |
+  Object ( [ value ] )
+
+  1. If NewTarget is neither undefined nor the active function, then
```

Comment:
I believe this test is wrong (although I could be wrong). If `NewTarget` is `O` and `active function` is `Object` (as is the case in this test), this should return a new object whose prototype is `NewTarget.prototype`, not `value`.